src/material/form-field/_form-field-outline-theme.scss

```diff
@@ -75,7 +75,7 @@ $mat-form-field-outline-dedupe: 0;
   scale($font-scale);
   width: 100% / $font-scale + $mat-form-field-outline-dedupe;
 
-  $mat-form-field-outline-dedupe: $mat-form-field-outline-dedupe + 0.00001 !global;
+  $mat-form-field-outline-dedupe: $mat-form-field-outline-dedupe + 0.00001;
```

Comment:
Do I miss something? Isn't that variable declared at the top-level of the file? (as per Sass docs that makes the variable global)

Did you check if the mixin actually still changes as expected?

Reply:
It's at the top level of the file, but it's still local to that file. My understanding is that `!global` is supposed to target variables that you don't have access to within the same file. I tested it out and the output looked identical.

Reply:
I think that does not matter. If the variable is at the top-level (even in the same file), then it is considered global, and non-top level assignments would need `!global` to update the value.

https://sass-lang.com/documentation/variables#shadowing

Reply:
It looks like this change will break the current behavior: https://www.sassmeister.com/gist/8961c3a4505e958d667a586eea64f914